static/CNAME

```diff
@@ -1 +0,0 @@
-preview.dogecoin.com

```

Comment:
was this deletion intentional?

Reply:
yes since this wasn't merged when i initially opened the pr it broke the preview i had on my repo since preview.dogecoin.com isn't pointing to my gh pages.